README.md

```diff
@@ -31,6 +31,7 @@ The following envioronment variables can be set:
 | SDX_VALIDATE_URL          | ``http://sdx-validate:5000/validate`` | URL of the ``sdx-validate`` service
 | SDX_RESPONSES_URL         | ``http://sdx-store:5000/responses``   | URL of the ``sdx-store`` service
 | SDX_COLLECT_SECRET        | _none_                                | Key for decrypting messages from queue, must be the same as used for ``sdx-receipt``
+| SDX_ENVIRONMENT           | ``sdx_development``                   | Name of the environment this service is running in
```

Comment:
I think this should be more explicit if the contents of this value are used exclusively for the DAP payload e.g. `DAP_SOURCE_NAME`

Reply:
I wrestled with this for a while, should this be specific to DAP or should I make an effort to make it generic in some way?  Sounds reasonable though, as I made it kind of generic 'sdx_prod/preprod' as opposed to just 'prod/preprod'.

I'll get that changed now